src/containers/ProfileEdit/EditForm/EditForm.tsx

```diff
@@ -10,7 +10,7 @@ import TextField from '@mui/material/TextField';
 import CircularProgress from '@mui/material/CircularProgress';
 import Button from '@mui/material/Button';
 import Avatar from '@mui/material/Avatar';
```

Comment:
p4;
하나로 묶을 수 있을 것 같습니다.
```suggestion
import {CircularProgress ,Button, Avatar} from '@mui/material';
```